temoa-uiuc/data_files/run_mga_scenario03.txt

```diff
@@ -8,15 +8,15 @@
 # Input File (Mandatory)
 # Input can be a .sqlite or .dat file
 # Both relative path and absolute path are accepted
---input=data_files/03_uiuc.sqlite
+--input=data_files/03_uiuc_mga.sqlite
```

Comment:
should 03_uiuc_mga be already a file in the repo or is this file created with the Snakefile?

Reply:
Snakemake creates the required files if they don't already exist by the rule "make_temoa_input"